docs/optimization_guide/nncf/ptq/code/ptq_torch.py

```diff
@@ -15,7 +15,8 @@ def transform_fn(data_item):
 #! [dataset]
 
 #! [quantization]
-model = ... # torch.nn.Module object
+import torch.nn as nn
+model = nn.Module() # torch.nn.Module object
```

Comment:
How about such an example: 

```suggestion
import torchvision
model = torchvision.models.resnet50(pretrained=True)
```
Would it be viable in this context then?

Reply:
I don't mind